app/routes/events.tsx

```diff
@@ -0,0 +1,31 @@
+import { useLoaderData } from "@remix-run/react";
+import { MetaFunction, json } from "@vercel/remix";
+
+import { EventsList } from "~/components/EventsList";
+import { Layout } from "~/components/Layout";
+import { createMeta } from "~/config";
+import { eventSchema } from "~/schemas";
+
+export const loader = async () => {
+	const events = (await import("../data/events.json")).default.map((event) => ({
```

Comment:
Is this typesafe? I don't recall using this syntax.
If not, since we're using `esModuleInterop` and unless you have plans to load this dynamically from some API or whatnot, maybe `import events from "../data/events.json"` could do the trick?

Reply:
It is type-safe, yeah. [dynamic `import()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import) is a full type-safe thing.

Reply:
That's awesome, I hadn't realized that, thanks!

Out of curiosity, why did you go with a dynamic `import()` vs. a regular static `import` here?